src/context.cpp

```diff
@@ -48,51 +51,94 @@ void set_context_event_address(object_store &store)
     store.insert(event_addr_idx, event_addr, true_obj, attribute::none);
 }
 
+// NOLINTBEGIN(cppcoreguidelines-avoid-const-or-ref-data-members)
+struct result_wrapper {
+    ddwaf_object &events;
+    ddwaf_object &actions;
+    ddwaf_object &duration;
+    ddwaf_object &timeout;
+    ddwaf_object &attributes;
+    ddwaf_object &keep;
```

Comment:
looks unused

Reply:
Yes, this will be used in the next PR (trace-tagging)